packages/e2e-tests/specs/editor/plugins/block-directory-add.test.js

```diff
@@ -166,7 +166,7 @@ const matchUrl = ( reqUrl, urls ) => {
 };
 
 describe( 'adding blocks from block directory', () => {
-	beforeEach( async () => {
+	beforeAll( async () => {
 		await createNewPost();
```

Comment:
For some reason the `adding blocks from block directory › Should be able to add (the first) block` spec started throwing the following error:

```
[Error: ReferenceError: wp is not defined
    at https://fake_url.com/block.js:2:26
    at https://fake_url.com/block.js:17:4]
[Error: ReferenceError: wp is not defined
    at https://fake_url.com/block.js:2:26
    at https://fake_url.com/block.js:17:4]
FAIL packages/e2e-tests/specs/editor/plugins/block-directory-add.test.js (106.681 s)
  ● adding blocks from block directory › Should be able to add (the first) block.

    TimeoutError: waiting for selector `.edit-post-layout` failed: timeout 30000ms exceeded

      at new WaitTask (../../node_modules/puppeteer-core/src/common/DOMWorld.ts:813:28)
      at DOMWorld.waitForSelectorInPage (../../node_modules/puppeteer-core/src/common/DOMWorld.ts:656:22)
      at Object.internalHandler.waitFor (../../node_modules/puppeteer-core/src/common/QueryHandler.ts:78:19)
      at DOMWorld.waitForSelector (../../node_modules/puppeteer-core/src/common/DOMWorld.ts:511:25)
      at Frame.waitForSelector (../../node_modules/puppeteer-core/src/common/FrameManager.ts:1273:47)
      at Page.page [as waitForSelector] (../../node_modules/puppeteer-core/src/common/Page.ts:[32](https://github.com/WordPress/gutenberg/actions/runs/3650930460/jobs/6167542890#step:6:33)10:29)
      at createNewPost (../e2e-test-utils/build/@wordpress/e2e-test-utils/src/create-new-post.js:[37](https://github.com/WordPress/gutenberg/actions/runs/3650930460/jobs/6167542890#step:6:38):8)
      at Object.<anonymous> (specs/editor/plugins/block-directory-add.test.js:176:3)
```

I haven't been able to figure out how this is related to my changes as I cannot reproduce locally, but the reason might be the request interception bleeding into the `post-new.php` request. No idea why it would start happening now. Anyway, letting both steps be tested on the same post fixes the issue, and doesn't degrade the value of the spec so it's OK to go with this fix IMO.

Pinging @StevenDufresne and @TimothyBJacobs (who I believe worked on most part of this spec) for insight on what might be going on here. 🙏

Reply:
Sorry, I don't think I can be of much help here. My work was mainly on the REST API side of things.

Reply:
Hey, I've noted this. Hopefully, I can take a look before the end of the week although a lot has changed in the Gutenberg project since we landed the original PR.

Reply:
Thanks for checking, @StevenDufresne! The tests are passing on this branch though due to [this change](https://github.com/WordPress/gutenberg/pull/46366/commits/2ee8335caa017306a8ed6fb50ae438a5027a58c1) I had to make. I've probably not made it clear enough in [my explanation above](https://github.com/WordPress/gutenberg/pull/46366#discussion_r1044210106) – sorry for that. The test should start failing in CI once the https://github.com/WordPress/gutenberg/pull/46366/commits/2ee8335caa017306a8ed6fb50ae438a5027a58c1 commit is reverted (currently the last one). Should I revert it on this branch so you can investigate it in CI?

Reply:
Ah, my mistake.

I think investigating would be investing more time than it's worth. If you want to be 100% safe, view the test run interactively and if it still adds the block to the document we are good. I would be pretty surprised if it passed without doing so though. 👍

Reply:
Gotcha. I did check via the interactive mode, and the block is correctly added to the canvas. 🙌 

/cc @tyxla, this one should be ready for a review if you have a spare cycle! 🙏

Reply:
Sounds good, thanks @WunderBart! I'll take a look tomorrow if I don't get to it today.